.github/workflows/black.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint code with Black
```

Comment:
We can check this before main workflow and ensure tests won't run unless lint tests pass

Reply:
Hi, it's an awesome suggestion @devkapilbansal. I tried but the trigger doesn't work! Can you help?

Reply:
> Hi, it's an awesome suggestion @devkapilbansal. I tried but the trigger doesn't work! Can you help?

Check @vj-codes's recent pr on mentorship system